pkg/virt-operator/resource/apply/apps.go

```diff
@@ -387,3 +398,25 @@ func (r *Reconciler) syncPodDisruptionBudgetForDeployment(deployment *appsv1.Dep
 
 	return nil
 }
+
+func getDesiredApiReplicas(clientset kubecli.KubevirtClient) (replicas int32, err error) {
+	nodeList, err := clientset.CoreV1().Nodes().List(context.Background(), metav1.ListOptions{})
+	if err != nil {
+		return 0, fmt.Errorf("failed to get number of nodes to determine virt-api replicas: %v", err)
+	}
+
+	nodesCount := len(nodeList.Items)
```

Comment:
```suggestion
	nodesCount := nodeList.Size()
```

Reply:
Will do, thanks!

Reply:
I'm sorry, I take it back, that's actually wrong :)
This returns some internal size which is not the number of element in the list.

Reply:
Urg, sorry!

Reply:
No need to be :) thanks for reviewing!!
It's indeed weird that a list type doesn't have a method to return its (actual) size..